doc/nrf/known_issues.rst

```diff
@@ -834,6 +834,17 @@ SoftDevice Controller
 
 .. rst-class:: v1-5-0
 
+DRGN-15465: Corrupted advertising data when :option:`CONFIG_BT_EXT_ADV` is set
+  Setting scan response data using legacy HCI commands on a build with extended advertising support corrupts parts of the advertising data.
```

Comment:
@rugeGerritsen @jori-nordic  This doesn't match my understanding of the issue.
Isn't it it setting scan response data for a legacy advertiser using extended HCI commands?

Reply:
Would this be better then:
Setting scan response data for a legacy advertiser on a build with extended advertising support corrupts parts of the advertising data.

Reply:
Yes, that seems correct.

Reply:
Shouldn't it rather be:
Setting scan response data for a legacy advertiser using extended advertising commands corrupts parts of the advertising data.
?
When setting scan response data on a build with extended advertising support using legacy commands works just fine.

Reply:
Yes but this part right there in hci_driver.c will ensure we only use the extended advertising commands from the zephyr host right ?
```c
	if (IS_ENABLED(CONFIG_BT_BROADCASTER)) {
		if (IS_ENABLED(CONFIG_BT_CTLR_ADV_EXT)) {
			err = sdc_support_ext_adv();
```

Reply:
what Im trying to say is that for all intents and purposes, this sentence correctly describes the SDC behavior when integrated with zephyr and when enabling ext adv.